### PR TITLE
EVG-5466 even more comprehensive query for timed out task job

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: units/crons.go [hunk 1]
```diff
@@ -228,14 +229,25 @@ func PopulateTaskMonitoring() amboy.QueueOperation {
...
+		}
+		for _, t := range tasks {
+			taskIDs[t.Id] = t.Execution
+		}
 
 		catcher := grip.NewBasicCatcher()
-		for _, t := range tasks {
-			catcher.Add(queue.Put(NewTaskExecutionMonitorJob(t.Id, t.Execution, 1)))
+		for id, execution := range taskIDs {
+			catcher.Add(queue.Put(NewTaskExecutionMonitorJob(id, execution, 1)))
```
[the remark this answers is not in the document]
Comment (john-m-liu) [reply units/crons.go]: added a 15 minute time to the ID

**Comment by unrealsolver** [comment]
lgtm

File: model/host/db.go [hunk 2]
```diff
@@ -487,6 +487,12 @@ func FindStaleRunningTasks(cutoff time.Duration) ([]task.Task, error) {
 			},
 		},
 	})
+	pipeline = append(pipeline, bson.M{
+		"$project": bson.M{
+			task.IdKey:        1,
+			task.ExecutionKey: 1,
+		},
+	})
```
Comment (tychoish) [inline model/host/db.go]: why not merge this into the above declaration? why do we need to do appens at all?

Comment (john-m-liu) [reply model/host/db.go]: I find that this is easier to read

**Pull request merged**